src/scripts/Update.ts

```diff
@@ -235,6 +235,21 @@ class Update implements Saveable {
                 console.error('[update] v0.5.2 - Couldn\'t start Aerodactyl Quest line..', ಠ_ಠ);
             }
         }
+        
+        if (this.isOlderVersion(this.saveVersion, '0.5.5')) {
+            try {
+                //Correct statistics
+                saveData.statistics.dungeonsCleared = Update.moveIndex(saveData.statistics.dungeonsCleared, 22, 34); // Petalburg Woods
+                saveData.statistics.dungeonsCleared = Update.moveIndex(saveData.statistics.dungeonsCleared, 30, 35); // New Mauville
+                saveData.statistics.dungeonsCleared = Update.moveIndex(saveData.statistics.dungeonsCleared, 56, 50); // Hall of Origin
+                saveData.statistics.dungeonsCleared = Update.moveIndex(saveData.statistics.dungeonsCleared, 33); // Hall of Origin
```

Comment:
This comment supposed to say sealed chamber?
No other issue as far as I can tell :+1:

Reply:
Sure is 🤦 
```suggestion
                saveData.statistics.dungeonsCleared = Update.moveIndex(saveData.statistics.dungeonsCleared, 33); // Sealed Chamber
```